inc/mlx90632.h

```diff
@@ -324,6 +363,30 @@ void mlx90632_set_emissivity(double value);
  */
 double mlx90632_get_emissivity(void);
 
+/** Trigger start of burst measurement for mlx90632
+ *
+ * Trigger a single measurement cycle and wait for data to be ready. It does not read anything, just triggers and completes.
+ * The SOB bit is set so that the complete measurement table is re-freshed.
+ * @note The SOB bit is cleared internally by the mlx90632 immediately after the measurment has started.
+ *
+ * @retval <0 Something failed. Check errno.h for more information
+ * @retval 0 New data is available and waiting to be processed
+ *
+ * @note This function is using usleep so it is blocking!
```

Comment:
```suggestion
 * @note This function is using usleep and msleep. Because of usleep it is blocking, while msleep implementation can have a thread switch!
  * In case both are blocking expect up to 2 second freeze of CPU in worse case scenario (depending on Refresh rate setting), so
  * you might also need to take care of Watch Dog.
```